packages/devtools_app/lib/src/debugger/debugger_screen.dart

```diff
@@ -193,8 +195,19 @@ class DebuggerScreenBodyState extends State<DebuggerScreenBody>
           initialFractions: const [0.40, 0.40, 0.20],
           minSizes: const [0.0, 0.0, 0.0],
           headers: <PreferredSizeWidget>[
-            const AreaPaneHeader(
-              title: Text(callStackTitle),
+            AreaPaneHeader(
+              title: const Text(callStackTitle),
+              rightActions: [
+                CopyToClipboardControl(
+                  dataProvider: () =>
```

Comment:
can you add a frame number (e.g. #0, #1, etc) to each frame's call stack display? You can use a for loop here instead of mapping, and append the index to the front of frame.callStackDisplay.

Reply:
I created a list and added the index to that instead as the code looked a little bit cleaner this way